tests/telemetry/test_telemetry_metrics.py

```diff
@@ -20,6 +20,7 @@ def _assert_metric(
 ):
     test_agent.telemetry_writer.periodic()
     events = test_agent.get_events()
+    # print(f"EVENTS EVETNTS {events=}")
```

Comment:
```suggestion
```

Reply:
Damnit. I thought I got all those. 😅 

Can you tell I got frustrated because we leak our external CI env into our tests, and then we have that funny fixture that re-instantiates the writer? 😛

Reply:
Deleted in [af51bd3](https://github.com/DataDog/dd-trace-py/pull/8948/commits/af51bd374a4aba7bc6393a2390d231fb62a54664).